db/kafka/scripts/topic-add.sh

```diff
@@ -105,5 +105,5 @@ export index
 if [ "$environment" = "local" ]; then
   docker exec -i tno-broker bash -c "/bin/kafka-topics --create --topic $topic --bootstrap-server $bootstrap --partitions $partitions --replication-factor $replication"
 else
-  oc rsh -n 9b301c-$environment tno-kafka-broker-$index bash -c "/bin/kafka-topics --create --topic $topic --bootstrap-server $bootstrap --partitions $partitions --replication-factor $replication"
+  oc rsh -n 9b301c-$environment kafka-broker-$index bash -c "/bin/kafka-topics --create --topic $topic --bootstrap-server $bootstrap --partitions $partitions --replication-factor $replication"
```

Comment:
Same issue here

Reply:
Pushed both changes and added a new file for Openshift deployments.